lib/components/easy-search-components.js

```diff
@@ -646,6 +646,24 @@ EasySearch.getComponentInstance = (function () {
 
         // Clear the selected items
         SearchVars.set(this.generateId(), { 'autosuggestSelected' : [] });
+      },
+      manageAutosuggestValues : function(callback) {
+        // Make sure callback is a function, if not return false - should this throw an error instead?
+        if ( !_.isFunction(callback) ) {
```

Comment:
better throw a Meteor.Error here and don't return anything. Otherwise looks fine